org.eclipse.corrosion/src/org/eclipse/corrosion/debug/DebugUtil.java

```diff
@@ -0,0 +1,43 @@
+/*********************************************************************
+ * Copyright (c) 2018 Red Hat Inc. and others.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *  Max Bureck (Fraunhofer FOKUS) - Initial implementation
+ *******************************************************************************/
+package org.eclipse.corrosion.debug;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.core.runtime.Platform;
+
+/**
+ * This class contains static helper methods and fields to be used for the
+ * Debuging functionality in Eclipse Corrosion
+ */
+class DebugUtil {
+
+	public static final boolean IS_WINDOWS = Platform.getOS().equals(Platform.OS_WIN32);
```

Comment:
I don't see it used any place else, so it should be private.

Reply:
Also, there is only one occurence of this, so it doesn't seem profitable to declare it and it could as well be directly inlined on line 40.

Reply:
Yes it should be private, but inlining would cause the evaluation on every call which seems to be unnecessary, especially since the OS won't change between calls.

Reply:
On the other hand, it retains the memory size of a boolean permanently for an operation that user runs not so frequently. So it's a matter of preference. Given that memory consumption is more visible than an extra string comparison on every run (not perceivable by end-user as cost of UI rendering >>> cost of a string compare), I'd favor the re-evaluation over the storage.
But that's nitpicking ;)